internal/venice-common/src/main/resources/avro/PartitionState/v17/PartitionState.avsc

```diff
@@ -235,6 +235,11 @@
       "doc": "The last offset consumed successfully in this partition from the version topic. This is used during resubscription when the Global RT DIV feature is enabled.",
       "type": "long",
       "default": -1
+    }, {
+      "name": "heartbeatLag",
```

Comment:
Instead of lag, wouldn't it be more useful to store the timestamp of the last heartbeat?